arm/cdn/customdomains.go

```diff
@@ -53,7 +53,7 @@ func NewCustomDomainsClientWithBaseURI(baseURI string, subscriptionID string) Cu
 // endpointName is name of the endpoint within the CDN profile profileName is
 // name of the CDN profile within the resource group resourceGroupName is
 // name of the resource group within the Azure subscription
-func (client CustomDomainsClient) Create(customDomainName string, customDomainProperties CustomDomainParameters, endpointName string, profileName string, resourceGroupName string) (result CustomDomain, ae error) {
+func (client CustomDomainsClient) Create(customDomainName string, customDomainProperties CustomDomainParameters, endpointName string, profileName string, resourceGroupName string) (result autorest.Response, ae error) {
```

Comment:
We should add documentation to these long-running operation methods (e.g. `Create` and `CreateSender`), indicating:

1. SDK will do the polling of this long running operation for you.
2. Returned `autorest.Response` contains the final HTTP response from the polling operation.

We _really_ need to make this situation clear as I stated in my response to the offline thread.